src/api/Project/mutations/createProject/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import ProjectModel from '@models/Project'
+import type { CreateProjectArgs, CreateProjectResponse } from '@models/Project'
+
+const resolver = async (
+  _: any,
+  args: CreateProjectArgs
+): Promise<CreateProjectResponse> => {
+  const { errors, project } = await ProjectModel.createProject(args)
+
+  if (errors.length > 0) {
+    return { errors }
+  }
+
+  return { project, errors }
```

Comment:
After discussing with Paul yesterday:
For GraphQL we’ll actually do:

```js
if (errors.length > 0) {
  return { status: 'FAILURE', errors }
}

return { status: 'SUCCESS', project }
```

But we’ll do that in another PR :)

Reply:
Yes, good idea 👍

Reply:
Or we could maybe do:

```js
if (errors.length > 0) {
  throw new Error('…') // Error only takes a string…
}
```

So I think we’d have to use `JSON.stringify(errors)` an then use `apollo-server`’s `formatError` to make it an array again… That sucks.